dataactvalidator/interfaces/interfaceHolder.py

```diff
@@ -8,10 +9,17 @@ class InterfaceHolder:
 
     def __init__(self):
         """ Create the interfaces """
-        self.jobDb = ValidatorJobTrackerInterface()
-        self.errorDb = ValidatorErrorInterface()
-        self.stagingDb = ValidatorStagingInterface()
-        self.validationDb = ValidatorValidationInterface()
+        if BaseInterface.interfaces is None:
+            self.jobDb = ValidatorJobTrackerInterface()
+            self.errorDb = ValidatorErrorInterface()
+            self.stagingDb = ValidatorStagingInterface()
+            self.validationDb = ValidatorValidationInterface()
+            BaseInterface.interfaces = self
```

Comment:
I'm confused. Both `dataactvalidator.interfaces.InterfaceHolder` and `dataactbroker.handlers.InterfaceHolder` are setting a global object, but they're setting it to different values. Is the idea that `dataactvalidator` can never be used with `dataactbroker`? Is there a way to implement this without requiring that implicit conflict?